crates/revm/src/inspector.rs

```diff
@@ -8,13 +8,15 @@ use auto_impl::auto_impl;
 pub mod customprinter;
 pub mod gas;
 pub mod noop;
+pub mod tracer_eip3155;
 
 /// All Inspectors implementations that revm has.
 pub mod inspectors {
     #[cfg(feature = "std")]
     pub use super::customprinter::CustomPrintTracer;
     pub use super::gas::GasInspector;
     pub use super::noop::NoOpInspector;
+    pub use super::tracer_eip3155::TracerEip3155;
```

Comment:
```suggestion
    #[cfg(feature = "std")]
    pub use super::tracer_eip3155::TracerEip3155;
```

Same comment as for mod